### PR TITLE
ble details

### PR DESCRIPTION
**Pull request by gngster**
elaborate on technology of tag

### CODE REVIEW

**Comment by gngster** [comment]
pls look at only the latest commit!!! the first commit contained contradictory points that didnt make sense

**Comment by Zhiyuan-Amos** [comment]
@gngster 

> pls look at only the latest commit!!! the first commit contained contradictory points that didnt make sense

You can squash your commit if there's some mistakes that you made in the earlier commits :P

Also, just fyi for your title post, you can add `fixes #4` (see my [example](https://github.com/IFS4205-2018-Sem1-Team1/first-report/pull/37)) so once this PR is merged, the corresponding issue will automatically be closed.

File: tool_assessment.md [hunk 1]
```diff
@@ -104,3 +104,13 @@ To scan our web application for vulnerabilities like XSS, we will use [OWASP ZAP
...
+
+As it would be architecturally less complex for us to implement a web application that communicates with the tag, we have decided to make use of the Web Bluetooth API which is written in Javascript. As it is still a work in progress right now, only certain platforms have Web Bluetooth fully implemented & supported in the browsers. The full status of its implementation can be found [here](https://github.com/WebBluetoothCG/web-bluetooth/blob/master/implementation-status.md).  Chrome OS, Mac and Android have it fully supported and implemented. Windows and Linux have it partially implemented and a flag must be enabled. 
+
+ As it is only possible to use the JustWorks method of BLE due to the I/O capabilities of the tag (no input and output capabilities), a MITM attack cannot be prevented. We would look at having some output capabiities for the tag. 
+ 
+ As we would be digitally signing the image to ensure that the image taken is of the patient, the private key would be stored in the tag, used to sign the image/ hash of the image and it can then be verfied by the public key the web app has.
```
Comment (Zhiyuan-Amos) [inline tool_assessment.md]: I think this part is too simplified. It's either we:
1. Remove this paragraph. This paragraph isn't a technology per se, and I think the above paragraphs you've described pros and cons of using BLE over web app. So this paragraph isn't too necessary.
1. Add a lot more information such as:
    1. Add the image that Prof Hugh drew on the board during our first meeting
    1. Explain the image
    1. Explain why we need to digitally sign

**Comment by gngster** [comment]
i listed out in steps instead because i feel it is much clearer than the diagram. also the diagram i need help understanding the use of a RAND? @Ley94 and yea ley can help me review see what i wrote makes sense not

**Comment by Zhiyuan-Amos** [comment]
Currently, the entire document is changed (see [Files changed](https://github.com/IFS4205-2018-Sem1-Team1/first-report/pull/38/files)). This makes it hard to review. I'm not sure what's the issue, but you can try rebasing your branch with `master`.

File: tool_assessment.md [hunk 2]
```diff
@@ -107,10 +107,20 @@ As we would need to uphold our security claims to our users, we will also consid
...
 
- As it is only possible to use the JustWorks method of BLE due to the I/O capabilities of the tag (no input and output capabilities), a MITM attack cannot be prevented. We would look at having some output capabiities for the tag. 
+Due to the  I/O capabilities of the tag (no input and output capabilities), it is only possible to use the [JustWorks](https://www.digikey.com/eewiki/display/Wireless/A+Basic+Introduction+to+BLE+Security) method of BLE which offers no way of verifying the devices taking part in the connection so a MITM attack cannot be prevented. We would look at having some output capabiities for the tag.
```
Comment (Zhiyuan-Amos) [inline tool_assessment.md]: "Due to the  I/O"

Extra whitespace

**Pull request merged**